psw/urts/parser/elfparser.cpp

```diff
@@ -856,6 +892,19 @@ bool ElfParser::update_global_data(const metadata_t *const metadata,
         return false;
     }
     *data_size = sizeof(global_data_t);
+
+    // Record the data and bss section address and size.
+    global_data_t *global_data = reinterpret_cast<global_data_t *>(data);
+    const ElfW(Ehdr) *elf_hdr =
+        reinterpret_cast<const ElfW(Ehdr) *>(m_start_addr);
+    const ElfW(Shdr) *shdr = get_section_by_name(elf_hdr, ELF_DATA);
+    global_data->data_offset = static_cast<sys_word_t>(shdr->sh_addr);
+    global_data->data_size = static_cast<sys_word_t>(shdr->sh_size);
+
+    shdr = get_section_by_name(elf_hdr, ELF_BSS);
+    global_data->bss_offset = static_cast<sys_word_t>(shdr->sh_addr);
+    global_data->bss_size = static_cast<sys_word_t>(shdr->sh_size);
+
```

Comment:
The structure `global_data_t` does not have the member data_offset data_size bss_offset bss_size

Reply:
Oh right, this got added for Asylo to support fork(). Removed here.